### PR TITLE
Feat/home

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Add landing page
- Add Dark mode

## Screenshot 
image

image

## Files to be ignored while reviewing

- package.json
- package-lock.json

## Preview Link:
https://deploy-preview-1--guessthisquiz.netlify.app/

### CODE REVIEW

**Review by saicharan-gouru — APPROVED** [review]
-UI looks so cool. Theme working fine.
-Good implementation of folder structure and code structure.
-Good that you have added description and screenshot for reference.
-Looks good to merge.

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
+import "./App.css";
+import { Home } from "./pages/index";
```
Comment (rohanmathur91) [inline src/App.js:2]: no need to mention /index

File: src/components/Button/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom";
+const Button = ({ buttonState, route }) => {
+  return (
+    <li>
+      <Link to={`/${route}`} className="nav-icon">
+        <button className="btn btn-ghost-primary">{buttonState}</button>
+      </Link>
```
Comment (rohanmathur91) [inline src/components/Button/Button.jsx:7]: why button, inside a link, keep it link

**Review by rohanmathur91 — APPROVED** [review]
- Good use of useLocation hook
- Code is also easily understandable and readable
- Added a few suggestions and sanity comments

File: src/App.js (hunk 1 above)
Comment (Aditi0505) [reply src/App.js:2]: Sure, I will remove that.

File: src/components/Button/Button.jsx (hunk 2 above)
Comment (Aditi0505) [reply src/components/Button/Button.jsx:7]: Sure, I will change this.

**Pull request merged**